ovos_plugin_common_play/ocp/gui.py

```diff
@@ -387,10 +387,15 @@ def handle_play_skill_featured_media(self, message):
         self.player.playlist.clear()
         self.player.media.replace(playlist)
         self["displaySuggestionBar"] = False
-        self._show_suggestion_disambiguation()
 
         self.manage_display("disambiguation")
 
+        # Model and page are heavy wait for them to load
+        # Calling event has no listners on first boot
+        # As page has never been loaded in GUI stack before
+        sleep(1.5)
```

Comment:
as discussed last time, please create a TODO to remove this sleep and make it event based using `self.bus.on`

the whole repo should be reviewed for more places where this happens

TODO comments make it easy to search and some IDEs even flag them

Reply:
refactored in #58 , was just adding the TODO comments but ended up getting a proof of concept implementation going